2-mandatory/3-stocks.js

```diff
@@ -34,7 +34,15 @@ const CLOSING_PRICES_LAST_5_DAYS_FOR_ALL_STOCKS = [
         Functions can help with this!
 */
```

Comment:
A couple of small points:
- think about indentation (similar comment to above)
- Again, this might be a good place to have a separate function that calculates the average for a single stock. Your solution still works - but breaking it down into functions can make your code a bit more readable 😄